tests/ballerina-integration-test/src/test/resources/http/httpservices/http-status-codes.bal

```diff
@@ -42,15 +42,23 @@ service<http:Service> differentStatusCodes bind {port : 9223} {
         path:"/createdWithBody"
     }
     sendCreatedWithBody (endpoint caller, http:Request req) {
-        _ = caller -> created(message = "Created Response");
+        _ = caller -> created("/newResourceURI", message = "Created Response");
```

Comment:
There should not be any spaces around `->` operator. Need to update the codes in below places as well.